federation/ldap/src/main/java/org/keycloak/storage/ldap/idm/query/internal/LDAPQuery.java

```diff
@@ -51,20 +51,19 @@ public class LDAPQuery implements AutoCloseable {
 
     private final LDAPStorageProvider ldapFedProvider;
 
-    private int offset;
```

Comment:
It's not used anywhere. `offset` is not supported in LDAPQuery. I confused me a bit earlier, so to avoid additional confusion I decided to remove the field completely.